io.parmentier.flatpak.streamdeck.yml

```diff
@@ -0,0 +1,67 @@
+app-id: io.parmentier.flatpak.streamdeck
```

Comment:
give the URL of the app upstream is https://timothycrosley.github.io/streamdeck-ui/ I don't think this app-id is appropriate.